codecov.yml

```diff
@@ -6,4 +6,7 @@ coverage:
         informational: true
     patch:
       default:
-        informational: true
\ No newline at end of file
+        informational: true
+# HACK: Ignore hardware module for now to avoid spammy messages
+include:
```

Comment:
Maybe "exclude" instead of "include"?

Reply:
Ohh dear :laughing:. It turns out it should be `ignore` instead.